gax/pom.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>gax</artifactId>
+  <version>2.7.1-SNAPSHOT</version><!-- {x-version-update:gax:current} -->
+  <packaging>jar</packaging>
+  <name>GAX (Google Api eXtensions) for Java (Core)</name>
+  <description>Google Api eXtensions for Java (Core)</description>
+
+  <parent>
+    <groupId>com.google.api</groupId>
+    <artifactId>gax-parent</artifactId>
+    <version>2.7.1-SNAPSHOT</version><!-- {x-version-update:gax:current} -->
+  </parent>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.api</groupId>
+      <artifactId>api-common</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.auth</groupId>
+      <artifactId>google-auth-library-credentials</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.threeten</groupId>
+      <artifactId>threetenbp</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.auth</groupId>
+      <artifactId>google-auth-library-oauth2-http</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.opencensus</groupId>
+      <artifactId>opencensus-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.auto.value</groupId>
+      <artifactId>auto-value-annotations</artifactId>
+      <scope>provided</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <resources>
+      <resource>
+        <includes>
+          <include>dependencies.properties</include>
+        </includes>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
```

Comment:
Q: does it correctly expand `${project.version}`? Should we define a value as a Maven property?

Reply:
Yes. There is a test that verifies this as well.